libs/owasp-diff-runner/src/main/java/ddf/lib/OwaspDiffRunner.java

```diff
@@ -145,9 +145,11 @@ private static String getMavenHome() throws OwaspDiffRunnerException {
           Arrays.stream(mavenVersionInfo.split(System.getProperty("line.separator")))
               .filter(str -> str.contains("Maven home:"))
               .findFirst()
-              .get()
-              .split("Maven home:")[1]
-              .trim();
+              .orElse(null);
```

Comment:
❓I believe the old code used to throw the `OwaspDiffRunnerException` when we did a `.get()` on something that was null so you can use `orElseThrwow()` instead of setting it to null.

Reply:
Very good idea. Makes the code look a lot cleaner too.

Reply:
I think you misunderstood my comment. What you had was fine but instead of doing the `orElse(null)` and returning null you should use `.orElseThrow` and throw the `OwaspDiffRunnerException`.